src/components/parameters/PropertyAnalyticsQuery-field.yaml

```
@@ -23,3 +23,4 @@ schema:
     - negotiation_margin
     - market_stress
     - construction_year
+    - market_stress
```

:hammer_and_wrench: issue:  `market_stress` est présent deux fois. Je suppose que c'est `market_stress_reliability` plutôt ?
```suggestion
    - market_stress_reliability
```